website/pages/docs/k8s/connect/ingress-gateways.mdx

```diff
@@ -5,4 +5,174 @@ sidebar_title: Ingress Gateways
+Values in this section override the values from the defaults stanza for the given ingress gateway with one exception:
+the annotations from the defaults stanza will be *appended* to any user-defined annotations defined in the gateways stanza rather than being overridden.
+Please refer to the ingress gateway configuration [documentation](/docs/k8s/helm#v-ingressgateways-defaults) for a detailed explanation of each option.
+
+-> *Note*: Make sure any ports that will be used as listeners in the ingress gateway's Consul config entry are included in the `ports` object for each gateway.
+
+## Deploying the helm chart
+
+Ensure you have the latest consul-helm chart and install Consul via helm using the following
+[guide](/docs/k8s/installation/overview#installing-consul) while being sure to provide the yaml configuration
+as previously discussed.
+
+## Configuring the gateway
+
+Now that Kubernetes has been updated, you must add the corresponding configuration to Consul. This requires you to use the Consul CLI.
+Configuring the ingress gateway requires:
+
+* Accessing the Consul agent
+* Submitting an Ingress Gateway configuration entry to Consul
+
+### Accessing the Consul agent
+
+You can access the Consul server directly from your host via `kubectl port-forward`, this is helpful for interacting with the Consul UI locally as well as validating connectivity of the application.
+
+```shell-session
+$ kubectl port-foward consul-server-0 8500 &
+```
+-> If TLS is enabled use port 8501
+
+-> Be sure the latest Consul binary is installed locally on your host.
+[https://releases.hashicorp.com/consul/](https://releases.hashicorp.com/consul/)
+
+-> If TLS is enabled set:
+```shell-session
+$ export CONSUL_HTTP_ADDR=https://localhost:8501
+```
+
+-> If ACLs are enabled set :
+```shell-session
+$ export CONSUL_HTTP_TOKEN=$(kubectl get secret consul-bootstrap-acl-token -o jsonpath={.data.token} | base64 -D)
+$ export CONSUL_HTTP_SSL_VERIFY=false
+```
+
+### Submitting an Ingress Gateway configuration entry
+
+Now that you have access to the Consul CLI you are able to submit an ingress gateway configuration entry.
+Here is an ingress gateway configuration.
+
+```shell
+Kind = "ingress-gateway"
+Name = "ingress-gateway"
+
+Listeners = [
+ {
+   Port = 8080
+   Protocol = "http"
+   Services = [
+     {
+       Name = "static-server"
+     }
+   ]
+ }
+]
+```
+
+Submit the ingress gateway configuration with the Consul CLI using this command.
+
+```shell-session
+$ consul config write ingress-gateway.hcl
+```
+
+-> You can confirm the ingress gateways have been configured as expected by looking at the Consul UI: http://localhost:8500/ui/dc1/services/ingress-gateway/
+
+-> If TLS is enabled use port 8501
+
+## Defining an Intention
+
+If ACLs are enabled, you must define an [intention](/docs/commands/intention) to allow the ingress gateway to access the upstream services defined in the config entries.
```

Comment:
```suggestion
If ACLs are enabled, you must define an [intention](/docs/connect/intentions) to allow the ingress gateway to access the upstream services defined in the config entry.

To create an intention that allows the ingress gateway to route to the service `static-server`, run:
$$$shell-session
$ consul intention create ingress-gateway static-server
$$$

```
(replace the dollar signs with backticks)

I think it makes more sense to link to the page about intentions in general rather than the `consul intention` cli command docs.

It's just one config entry at this point I think.